test/e2e/framework/internal/unittests/cleanup/cleanup_test.go

```diff
@@ -49,7 +49,7 @@ import (
 //
 // This must be line #50.
 
-var _ = ginkgo.Describe("framework", func() {
+var _ = ginkgo.Describe("e2e", func() {
```

Comment:
This gets changed to make it more obvious that this is unrelated to the "framework" in the annotated code location.